solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -17,20 +17,33 @@
 //! The bridge to ethereum config
 
 pub const SLOTS_PER_EPOCH: u32 = snowbridge_pallet_ethereum_client::config::SLOTS_PER_EPOCH as u32;
-
+#[cfg(not(test))]
+use crate::EthereumBeaconClient;
+use frame_support::weights::ConstantMultiplier;
+
+use sp_core::H160;
+use sp_core::{ConstU32, ConstU8};
+#[cfg(not(feature = "runtime-benchmarks"))]
+use tp_bridge::symbiotic_message_processor::SymbioticMessageProcessor;
 use {
     crate::{
-        parameter_types, weights, xcm_config::UniversalLocation, AggregateMessageOrigin, Balance,
-        Balances, EthereumOutboundQueue, EthereumSystem, FixedU128, GetAggregateMessageOrigin,
-        Keccak256, MessageQueue, Runtime, RuntimeEvent, TreasuryAccount, WeightToFee, UNITS,
+        parameter_types, weights, xcm_config, xcm_config::UniversalLocation,
+        AggregateMessageOrigin, Balance, Balances, EthereumInboundQueue, EthereumOutboundQueue,
+        EthereumSystem, FixedU128, GetAggregateMessageOrigin, Keccak256, MessageQueue, Runtime,
+        RuntimeEvent, TransactionByteFee, TreasuryAccount, WeightToFee, UNITS,
     },
     dancelight_runtime_constants::snowbridge::EthereumLocation,
     pallet_xcm::EnsureXcm,
     snowbridge_beacon_primitives::{Fork, ForkVersions},
     snowbridge_core::{gwei, meth, AllowSiblingsOnly, PricingParameters, Rewards},
-    sp_core::{ConstU32, ConstU8},
+    tp_bridge::{DoNothingConvertMessage, DoNothingRouter},
 };
 
+// Ethereum Bridge
+parameter_types! {
+    pub storage EthereumGatewayAddress: H160 = H160(hex_literal::hex!("EDa338E4dC46038493b885327842fD3E301CaB39"));
```

Comment:
What is this storage modifier? It means that this is stored in storage and can be modified by root?

Reply:
it's the first time that I see it too

Reply:
The storage modifier means following:
1. From name of the variable a key is derived. Formula for derivation is: `twox_128(":" ++ NAME ++ ":")`
2. If there is a value under the key, it is returned in response to `get()`
3. If not, value specified in the assignment `H160(hex_literal::hex!("EDa338E4dC46038493b885327842fD3E301CaB39"))` is returned.

Reply:
Also, yes since only root can write to arbitrary key, this is modified by root.

Reply:
does this need to be in storage? that is very interesting, so it means it can be changed without a runtime upgrade right?

Reply:
yes. That is the advantage. We do not need runtime upgrade. Root can just change the storage.

Reply:
*disregard*